draft/2024-11-27-this-week-in-rust.md

```diff
@@ -39,6 +39,8 @@ and just ask the editors to select the category.
 
 ### Observations/Thoughts
 
+* [Aligning with rust's Guiding Principle: my path to finding my 'Why' ](http://localhost:1313/posts/rust_guiding_principle/)
+
```

Comment:
Url contains localhost!
localhost:1313

Reply:
Oh that's embarrassing, sorry for that... That's what i get for using `conceallevel=2` :dagger: